ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -51,6 +52,9 @@
  */
 /*package*/ class NativeAnimatedNodesManager implements EventDispatcherListener {
 
+  // Used to avoid allocating a new array on every frame in runUpdates.
+  private final static List<AnimatedNode> sRunUpdateNodeList = new ArrayList<>();
```

Comment:
Both would work, having it as a static ivar avoids having multiple copies in case there are multiple instances of the animated module (like in exponent). The array gets cleaned up after each invocation of `runUpdates` and this always runs from the same thread so it is safe.

If you feel it should be a normal ivar I don't really mind, just saves a few bytes in rare situations.